src/i18n/strings/en_EN.json

```diff
@@ -428,6 +428,8 @@
     "Stops ignoring a user, showing their messages going forward": "Stops ignoring a user, showing their messages going forward",
     "Unignored user": "Unignored user",
     "You are no longer ignoring %(userId)s": "You are no longer ignoring %(userId)s",
+    "Ignored room": "Ignored room",
```

Comment:
Note that we're not ignoring the room, only the invites from this room!

Reply:
...actually, you are right, we may be ignoring the room entirely. But to do so, the slash command needs to leave the room immediately, no?